tests/test_cmd2.py

```diff
@@ -1368,6 +1368,30 @@ def test_select_invalid_option(select_app):
     # And verify the expected output to stdout
     assert out == expected
 
+def test_select_invalid_option_too_small(select_app):
+    # Mock out the input call so we don't actually wait for a user's response on stdin
+    m = mock.MagicMock(name='input')
+    # If side_effect is an iterable then each call to the mock will return the next value from the iterable.
+    m.side_effect = ['0', '1']  # First pass and invalid selection, then pass a valid one
```

Comment:
Change to "First pass an"
Same with line 1351

Reply:
Done